docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,13 +1,28 @@
-= Mequal
+= Mequal - Manifest Evaluation & Quality
```

Comment:
Can you use em dash?

https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/

Sorry for being picky, but I really love these 😄

Reply:
Done :)